i18n/en.json

```diff
@@ -256,7 +256,7 @@
   "admin.billing.company_info.numEmployees": "Number of employees (optional)",
   "admin.billing.company_info.title": "Company Information",
   "admin.billing.company_info.zipcode": "Zip/Postal Code",
-  "admin.billing.history.allPaymentsShowHere": "All of your monthly payments will show here",
+  "admin.billing.history.allPaymentsShowHere": "All of your invoices will be shown here",
```

Comment:
Should we also change the default message in the component that uses this? It's still using the old message: https://github.com/mattermost/mattermost-webapp/blob/b878ef85c23a7acf76cd388e3a92c085ad107f92/components/admin_console/billing/billing_history.tsx#L87